addons/godot-behavior-tree-plugin/tick.gd

```diff
@@ -15,18 +15,18 @@ func _init():
 	actor = null
 	blackboard = []
 
-func openNode(node):
+func openNode(_node):
```

Comment:
Does this actually affect any functionality?

Reply:
No, it does absolutelly nothing. Its just to stop the warning flooding.